rest-dashboard.go

```diff
@@ -170,6 +171,8 @@ type FoundBoard struct {
 // only starred dashboards and only for tags (logical OR applied to multiple tags).
 //
 // Reflects GET /api/search API call.
+// Deprecated: This interface does not allow for API extension and is out of date.
+// Please use SearchWithParams(WithSearchType(SearchTypeDashboard))
 func (r *Client) SearchDashboards(ctx context.Context, query string, starred bool, tags ...string) ([]FoundBoard, error) {
```

Comment:
Would it be possible to call SearchWithParams() from this function so that we wouldn't reimplement the same functionality twice?

Reply:
Certainly so. It was my intention to add it as a next step. Will add to this PR instead.

Reply:
Added in https://github.com/grafana-tools/sdk/pull/86/commits/b86aa5c380eeeb9fb0b74f668f5cb042489e8a26